code/datums/autolathe/engineering_vr.dm

```diff
@@ -0,0 +1,22 @@
+// Lets allow autolathes to build the basic tech 1 parts shall we?
```

Comment:
Science can already make infinite autolathes/protolathes, but I can see how giving cargo the means to churn them out might be an issue if they just stick one in the hall.